### ISSUE TITLE
results are skewed in jQuery example

### ISSUE THREAD

**Issue by r3wt** [jonmiles/react-performance-tests#1]
look at this pathetic code:

``` js
var $wrapper = $('<div></div>');
for (i = 0; i < data.length; i++) {
    $wrapper
        .append($('<div class="item" style="background-color:' + data[i].color + ';"></div>')
            .append('<label>' + data[i].val + '%</label>')
        );
}
this.$element.empty().append($wrapper);
```

though ugly, this would probably be the fastest possible implementation with jQuery:

``` js
var $wrapper = ['<div>'];
for (i = 0; i < data.length; i++) {
    $wrapper
        .push('<div class="item" style="background-color:')
        .push(data[i].color)
        .push(';"><label>')
        .push(data[i].val)
        .push('%</label>')
        .push('</div>')
}
this.$element.get(0).innerHtml = $wrapper.push('</div>').join('');
```

**Comment by jonmiles** [comment]
Thanks, that was useful.  You obviously didn't read the README.

**Comment by r3wt** [comment]
actually, its trivial to perform a vdiff on the object state as well, and only update the changed elements, further improving the performance. jQuery makes this painless as hell.

**Comment by mattjcowan** [comment]
Hey Jonathan, I realize you closed this 'issue', and I did go over the ReadMe, but I think that in general, we need to be fair to jQuery and common best practices; otherwise, it's really comparing apples and oranges. From your ReadMe, I gather you really are trying to achieve that objective, while keeping the implementation very simple of course, as a stated objective.

I made a few very simple/quick updates, I can submit a pull-request, a dozen lines or so. Thanks for putting this together by the way, it's helpful, and I think we can make it an even more helpful test by using some common jQuery best practices to make it a fair comparison.
